Initial Commit For Review

aReNGee opened the pull request:
Raining Kids added

Commit:
Initial commit, Game Proposal

aReNGee committed:
Added game proposal to readme

Commit:
Initial Commit of Project

Made by Lucas Kanabe and Ivan Boyko

aReNGee committed:
Added missing group members

Commit:
Initial Setup of Kids, pending assets

Commit:
Initial Setup of assets, pending kids

Commit:
Merge remote-tracking branch 'origin/Kanabe_Boyko' into Kanabe_Boyko

Commit:
Adjusting gitignore

Commit:
Added raining kids

k-kruusi commented on ItsRainingKids/ItsRainingKids/GameScene.swift:
```diff
@@ -0,0 +1,58 @@
+        //print(kids.last?.position.x, " ", kids.last?.position.y)
+    }
+    
+    @objc func CreateNewKid(){
```
I suspect this would eventually overrun the stack. Essentially every subsequent timer you make keeps a reference to the the function it was called in because that's it scope, and it would be deleted before it finished if it didn't. Since it keeps making new timers, increasing the size of this function every time its called. Instead what you want is: One timer that calls that function on a random interval. 

Although you weren't taught these kind of timers yet, you'll also need to invalidate them when they finish or they're never deleted.

k-kruusi commented on ItsRainingKids/ItsRainingKids/Kid.swift:
```diff
@@ -0,0 +1,96 @@
+{
+    var type: KidType
+    
+    let unitVector = CGPoint(x: 0, y:-5)
```
Not a unit vector.

k-kruusi reviewed (changes requested):
Good Start

ObiWanKanabe committed:
Added Trampoline and Kid bouncing

Trampoline class creation. Kids will bounce off of trampoline when
caught, score will imcrement, and kids will be deleted from the game
scene when out of the frame.

ObiWanKanabe committed:
Added menu scene and score reset
